### ISSUE TITLE
About falling of learning rate

### ISSUE THREAD

**Issue by Ugness** [Ugness/PiCANet-Implementation#17]
Hi,  @Ugness , I intergrate  your flie of measure.py and train.py ，but  I don't change the file of network.py . now , I set the value of batch_size is 2, at the first falling of learning rate,my  train loss can  falling .but after that,althought my learning rate falling ,my train loss never falling. And , I test my model on PASCAL-S ,the best value of MAE is 0.1243.could you help me and sovle this problem?

_Originally posted by @RaoHaobo in https://github.com/Ugness/PiCANet-Implementation/issues/9#issuecomment-478973294_
